features/facebookImport/src/views/ministories/story.jsx

```diff
@@ -23,10 +32,15 @@ class Story extends Component {
         return true;
     }
 
+    hasDetails() {
+        if (this._renderDetails) return true;
+        return false;
+    }
+
     render() {
-        if (this.props.mode === Story.MODES.DETAILS)
-            return this.renderDetails();
-        return this.renderSummary();
+        if (this.mode === Story.MODES.DETAILS && this._renderDetails)
```

Comment:
So there's actually an enum, can't you use it above?

Reply:
At least at the time I'm reading this, it seems to be used consistently above.